src/engine/source/vdscanner/testtools/rocksdb/main.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Wazuh Vulnerability scanner
+ * Copyright (C) 2015, Wazuh Inc.
+ * Nov 13, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <iostream>
+
+#include <httplib.h>
+
+#include "base/utils/rocksDBWrapper.hpp"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/idl.h"
+
+#include "argsParser.hpp"
+
+namespace json
+{
+    const auto OBJECT_OPEN = "{\"";
+    const auto OBJECT_CLOSE = "\"}";
+    const auto ARRAY_OPEN = "[";
+    const auto ARRAY_CLOSE = "]";
+    const auto COMMA = "\",";
+    const auto COLON = "\":\"";
```

Comment:
I was thinking about it, it will be best to parse it using nlohmann

Reply:
Added